From d8faa0c06b02bb5e58872ae180873cc85a151ca1 Mon Sep 17 00:00:00 2001
From: Charles-IV <cthefourth@gmail.com>
Date: Fri, 18 May 2018 10:10:28 +0100
Subject: [PATCH 165/249] wiki: signing_builds: include cm-14.1 documentation

This just saves time searching around on gerrit for those of us still building LineageOS 14.

Change-Id: Ie7f0df21d717b3f5d6f4edbf9150d138bedc9176
---
 pages/signing_builds.md | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/pages/signing_builds.md b/pages/signing_builds.md
index d42a4c3..c60c802 100644
--- a/pages/signing_builds.md
+++ b/pages/signing_builds.md
@@ -67,17 +67,24 @@ on all packages at every boot. Install them for as little time as possible." %}
 
 You can set up your own migration builds by running:
 
+LineageOS 15.1:
 ```
 repopick -f 192655 -P vendor/lineage
 repopick -f 192656 -P frameworks/base
 ```
 
+LineageOS 14.1:
+```
+repopick 156047 162144
+```
+
 Then, follow the [instructions to generate an install package](#generating-an-install-package).
 
 #### Going back
 
 After installing the migration build, you can switch back to building normal builds:
 
+LineageOS 15.1:
 ```
 cd vendor/lineage
 git reset --hard github/lineage-15.1
@@ -86,6 +93,15 @@ cd frameworks/base
 git reset --hard github/lineage-15.1
 ```
 
+LineageOS 14.1:
+```
+cd vendor/cm
+git reset --hard github/cm-14.1
+croot
+cd frameworks/base
+git reset --hard github/cm-14.1
+```
+
 ### Using a script
 
 You can also use a script or small flashable zip designed to be run once, before installing
-- 
2.17.1

